src/jvm/test/org/jetbrains/kotlinx/lincheck_test/representation/RunConcurrentRepresentationTests.kt

```diff
@@ -177,6 +177,7 @@ class VariableReadWriteRunConcurrentRepresentationTest : BaseRunConcurrentRepres
     }
 }
 
+@IgnoreInTraceDebuggerMode // TODO investigate (Evgeniy Moiseenko)
```

Comment:
Can we not ignore but have a different output file when `inTraceDebuggerMode == true`?

Reply:
No, because of the reasons mentioned above by @eupp.